web/src/compose_setup.js

```diff
@@ -70,6 +76,11 @@ export function initialize() {
         } else {
             $("#compose_close").attr("data-tooltip-template-id", "compose_close_tooltip_template");
         }
+        if (compose_text_length > 0) {
```

Comment:
Let's add a blank line and then comment line just before this, noting that "The poll widget requires an empty compose box."